Funky-Chickens/bangazonAPI
# products faker data (el)

I made the following changes:
add products and product types faker-creator functions in the products-db.js file, because we need to populate those into the DB.
  write two functions to insert them into the DB tables creates

Steps to test:
fetch and then run `node build-db` on the terminal line.   Open the database browser to see if the tables have been populated for products and product types.

[(link to feature ticket)](https://github.com/Funky-Chickens/bangazonAPI/issues/16)
